Demo/UITests/BraintreeDropIn_UITests.swift

```diff
@@ -180,6 +180,61 @@ class BraintreeDropIn_CardDisabled_UITests: XCTestCase {
     }
 }
 
+class BraintreeDropIn_CardLogosDisabled_UITests: XCTestCase {
```

Comment:
There is class for disabled and enabled because apparently the `app.launchArguments` need to be appended in setup instead of the test method. I guess that makes sense but would have been nice to have both tests in a single class, but that's not possible.